draft-ietf-cbor-7049bis.md

```diff
@@ -1539,32 +1539,32 @@ example:
   If NaN is an allowed value, the protocol needs to pick a single
   representation, for example 0xf97e00.
 * If a protocol includes a field that can express integers larger than
-  2^64 using tag 2 ({{bignums}}), the protocol's canonicalization
+  2^64 using tag 2 ({{bignums}}), the protocol's deterministic encoding
   needs to specify whether small integers are expressed using the tag
   or major types 0 and 1.
 * A protocol might give encoders the choice of representing a URL as
   either a text string or, using {{encodedtext}}, tag 32 containing a
-  text string. This protocol's canonicalization needs to either
+  text string. This protocol's deterministic encoding needs to either
   require that the tag is present or require that it's absent, not
   allow either one.
 
 ### Length-first map key ordering
 
-The core canonicalization requirements sort map keys in a different
+The core deterministic encoding requirements sort map keys in a different
 order from the one suggested by {{?RFC7049}}. Protocols that need to
```

Comment:
Here we need to refer to what is called "canonicalization" in 7049.

And we probably needs to explain why the term is different now.

Meta-comment: Before long, people will call the old ordering "canonical" and the new one "deterministic".  Then some PHBs will write "canonical" in their requirements and get the old ordering.  Can we prevent (er, work a bit against) that scenario?